ext/gd/config.m4

```diff
@@ -138,7 +138,8 @@ dnl
 if test "$PHP_GD" != "no"; then
 
   if test "$PHP_EXTERNAL_GD" = "no"; then
-    GD_CFLAGS=""
+    dnl Disable strict prototypes as GD takes advantages of variadic function signatures for function pointers.
```

Comment:
Can gd be fixed somehow too?

Reply:
No, because libgd does that and not the PHP implementation.